control-plane/pkg/reconciler/consumergroup/consumergroup.go

```diff
@@ -33,6 +34,14 @@ import (
 	kafkainternalslisters "knative.dev/eventing-kafka-broker/control-plane/pkg/client/internals/kafka/listers/eventing/v1alpha1"
 )
 
+const (
+	// KafkaConditionConsumerGroupReady has status True when the KafkaChannel has created a consumer group.
+	KafkaConditionConsumerGroupReady apis.ConditionType = "ConsumerGroupReady"
+
+	// KafkaConditionConsumerGroupNotReady has status False when the KafkaChannel has failed to create a consumer group.
+	KafkaConditionConsumerGroupNotReady apis.ConditionType = "ConsumerGroupNotReady"
+)
```

Comment:
I think we should have only one condition `ConsumerGroup` (or something like that) and then we can mark that condition with true or false based on the state or errors we get.

Conditions like this one should be marked as true to mark the entire KafkaSource as ready, so `ConsumerGroupNotReady` won't be true in this case.